packages/css/src/plugins/sort-atomic-style-sheet.ts

```diff
@@ -101,10 +109,6 @@ export const sortAtomicStyleSheet = (config: {
       }
 
       root.nodes = [...catchAll, ...rules, ...atRules.map((atRule) => atRule.node)];
-
-      if (sortShorthandEnabled) {
-        sortShorthandDeclarations(root.nodes);
-      }
```

Comment:
Got it, so it was just my bad this was ordered like this.

Reply:
it was my bad as well 😅 lots of subtle behaviour quirks and bugs i didn't spot even from working on it for hours